src/content/1.7/functors.tex

```diff
@@ -206,6 +206,8 @@ \subsection{Equational Reasoning}
   fmap g Nothing
 = { definition of fmap }
   fmap g (fmap f Nothing)
+= { definition of composition }
```

Comment:
This snippet seems to be out of scope of this PR, is it intended ?

Reply:
It was intended, but you're right that it's out of scope - I'll remove it.

Also thanks for the reminder, I completely forgot about this PR. I'll fix it this week.

Reply:
no problem :)